packages/react-ui-core/src/ToggleButton/__tests__/ToggleButton-test.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import ThemedToggleButton from '../ToggleButton'
+import theme from './mocks/theme'
+
+const ToggleButton = ThemedToggleButton.WrappedComponent
+
+describe('ToggleButton', () => {
+  it('renders', () => {
+    const props = {
+      theme,
+      className: theme.ToggleButton,
+      children: <a href="#">Toggle</a>,
+    }
+
+    const wrapper = shallow(<ToggleButton {...props} />)
+    expect(wrapper.find('[data-tid="toggleButton"]').exists()).toBeTruthy()
```

Comment:
You seem to be doing: 
```
wrapper.find('[data-tid="toggleButton"]')
```
all over this file.  I think you can just use the wrapper without the find which should give you the root.

Reply:
Fixed